src/query/api/v1/handler/namespace/get.go

```diff
@@ -71,10 +72,15 @@ func NewGetHandler(
 	}
 }
 
-func (h *GetHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+func (h *GetHandler) ServeHTTP(
+	svc handleroptions.ServiceNameAndDefaults,
```

Comment:
Doesn't `GetHandler` need to implement `http.Handler`?

Reply:
The convention we use in the other namespace handlers is to have this signature and wrap them, so just pattern matching that: https://github.com/m3db/m3/blob/2cc61a79909d6b3e18ffab35846a2e428334b637/src/query/api/v1/handler/namespace/common.go#L125-L155.

Reply:
I just realized though that the `Delete` handler should be updated to be wrapped as well, will add that on.